pandas/tests/io/formats/test_to_html.py

```diff
@@ -16,6 +16,8 @@
 
 import pandas.io.formats.format as fmt
 
+pytestmark = pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
is this actually filtering out all warnings?

that's the thing, how do we know this is not warning when it shouldn't be?

Reply:
yes that filters the generic warnings but the tests added, `test_future_warning` and `test_no_future_warning` explicitly test for the case where the warnings are generated as expected and no warnings are generated for the regular `df.to_html()` call.

Reply:
Can you make the filter a bit more specific? (eg match the beginning of the message) Just to ensure we only ignore this warning, and not accidentally suppress warnings we should see.